lib/internal/v8_prof_polyfill.js

```diff
@@ -36,43 +38,14 @@ if (module.id === 'internal/v8_prof_polyfill') return;
 
 // Node polyfill
 const fs = require('fs');
-const cp = require('child_process');
-const os = {
-  system: function(name, args) {
```

Comment:
Why are these removed? there are here to tidy up the profile for Linux and to address https://github.com/nodejs/node/issues/29804

Reply:
I've put them back, I couldn't find a way of adding a test that would fail when I delete those lines though.